sonar-plugin-api/src/main/java/org/sonar/api/issue/Issue.java

```diff
@@ -186,12 +195,18 @@
   boolean isNew();
 
   /**
-   * @since 5.0
+   * @deprecated since 5.5, replaced by {@link #effort()}
    */
-  @CheckForNull
+  @Deprecated
   Duration debt();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)